mtls/mtls.py

```diff
@@ -900,21 +953,56 @@ def add_user(self, fingerprint, is_admin=False):
         else:
             _type = "User"
         click.secho(
-            "Added {_type}: {fingerprint}".format(
-                fingerprint=fingerprint, _type=_type
-            ),
+            f"Added {_type}: {fingerprint}",
             fg="green",
         )
 
     def remove_user(self, fingerprint, is_admin=False):
         msg = "Signing Remove User Request..."
-        payload = {
-            "type": "ADMIN" if is_admin else "USER",
-            "fingerprint": fingerprint,
-            "signature": str(self.gen_sig(fingerprint.encode("UTF-8"), msg)),
-        }
-        response = self.send_request(payload, method="post")
-        response = response.json()
+        payload = {}
+        if is_admin:
+            payload["admin"] = True
+
+        if payload:
```

Comment:
What is this checking? the presence of the admin key?

Reply:
No, this is for if the user being modified is an admin or not.

Reply:
then shouldn't you use `is_admin`?

Reply:
It adds the admin flag when the user denotes so via a cli flag. That's what is_admin is defaulted to false. But when set by the user it sets it to true and adds the key/value pair. 

You could in theory always pass the admin status I guess. In future versions of the API the distinction won't matter as we won't be tied to PGP keyrings for admin vs user.